BridgeCareApp/BridgeCare/DataAccessLayer/AttributesByYear.cs

```diff
@@ -0,0 +1,112 @@
+﻿using BridgeCare.ApplicationLog;
+using BridgeCare.Interfaces;
+using BridgeCare.Models;
+using BridgeCare.Utility;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Linq;
+
+namespace BridgeCare.DataAccessLayer
+{
+    public class AttributesByYear : IAttributesByYear
+    {
+        private readonly BridgeCareContext db;
+
+        public AttributesByYear(BridgeCareContext context)
+        {
+            db = context ?? throw new ArgumentNullException(nameof(context));
+        }
+
+        public List<AttributeByYearModel> GetProjectedAttributes(SimulatedSegmentAddressModel segmentAddressModel, BridgeCareContext db)
```

Comment:
Why is the word "Address" in the name of the class/parameter?

Reply:
the 3 keys, or...Ids , or address needed to construct a query into a simulation results table and get data for each bridge. Need to construct table and query by section

var selectStatement = String.Format("SELECT * FROM SIMULATION_{0}_{1} WHERE SectionID={2}",
                segmentAddressModel.NetworkId, segmentAddressModel.SimulationId, segmentAddressModel.SectionId); 

how about Ids instead?